perfkitbenchmarker/configs/storage_utility.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import math
+import numpy
+
+
+class SQLProviderError(Exception):
```

Comment:
This sounds like the provider experienced an error. A better name is perhaps: "InvalidProviderError".

Also, this utility should be independent of "SQL provider", as you can see from the documentation, storage capability is decided by the block storage products (EBS in AWS, and Persistent Disk in GCP).